pom.xml

```diff
@@ -202,6 +208,12 @@
       <groupId>org.kohsuke.stapler</groupId>
       <artifactId>stapler-groovy</artifactId>
       <version>1756.vec9071748061</version>
+      <exclusions>
+        <exclusion>
+          <groupId>jakarta.annotation</groupId>
+          <artifactId>jakarta.annotation-api</artifactId>
+        </exclusion>
+      </exclusions>
```

Comment:
Was pulling in a Jakarta import version of this API which conflicts with that of Jetty.